macros/WCTE.mac

```diff
@@ -93,6 +97,10 @@
 /DarkRate/SetDarkLow 0
 /DarkRate/SetDarkWindow 4000
 
+# Neutron Capture Commands
+/WCSim/SaveCapture true
+/physics_engine/neutron/timeLimit 1000000 ns
+
```

Comment:
Only use with the AmBe generator? Also `/physics_engine/neutron/timeLimit` is for disabling simulation of unwanted neutrons, is this what you want?

Reply:
I think that saving the neutrons is needed in other studies.
`/physics_engine/neutron/timeLimit` is also needed. If I do not set it, I just see nCapture happening in ~5% of the events. Whereas when I set it, I see nCapture in ~95% of the events.

Reply:
Okay from geant4 [source code](https://github.com/Geant4/geant4/blob/v10.3.3/source/physics_lists/constructors/limiters/src/G4NeutronTrackingCut.cc), the default neutron tracking timing cut is 10micro second, so you set to 1000 to should make a difference. I wonder whether we will get 100% capture if we completely deactivate the process by using
```
/process/inactivate nKiller
```